Tests/SwiftSyntaxMacroExpansionTest/MacroExpansionContextTests.swift

```diff
@@ -0,0 +1,73 @@
+  //===----------------------------------------------------------------------===//
+  //
+  // This source file is part of the Swift.org open source project
+  //
+  // Copyright (c) 2014 - 2023 Apple Inc. and the Swift project authors
+  // Licensed under Apache License v2.0 with Runtime Library Exception
+  //
+  // See https://swift.org/LICENSE.txt for license information
+  // See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+  //
+  //===----------------------------------------------------------------------===//
+
+  //==========================================================================//
+  // IMPORTANT: The macros defined in this file are intended to test the      //
+  // behavior of MacroSystem. Many of them do not serve as good examples of   //
+  // how macros should be written. In particular, they often lack error       //
+  // handling because it is not needed in the few test cases in which these   //
+  // macros are invoked.                                                      //
+  //==========================================================================//
+
+import SwiftSyntax
+import SwiftDiagnostics
+@_spi(Diagnostics) import SwiftSyntaxBuilder
+import SwiftSyntaxMacros
+import SwiftSyntaxMacroExpansion
+import SwiftSyntaxMacrosTestSupport
+import XCTest
+
+//private class TestMacroExpansionContext: MacroExpansionContext {
```

Comment:
TODO: Remove the commented out test expansion context, basic context worked out fine.